src/panels/config/lovelace/dashboards/ha-config-lovelace-dashboards.ts

```diff
@@ -279,9 +279,15 @@ export class HaConfigLovelaceDashboards extends LitElement {
       removeDashboard: async () => {
         if (
           !(await showConfirmationDialog(this, {
+            title: this.hass!.localize(
+              "ui.panel.config.lovelace.dashboards.confirm_delete_title",
+              "dashboard_title",
+              dashboard!.title
+            ),
```

Comment:
```suggestion
            title: this.hass!.localize(
              "ui.panel.config.lovelace.dashboards.confirm_delete_title",
              {dashboard_title: dashboard!.title}
            ),
```

Reply:
Update syntax to use object for localize arguments

Reply:
Thanks for this suggestion. First time contributing and I'd found localize called like this elsewhere so assumed that was how it was meant to be called.

Reply:
No problem! :-) Yes, we are moving to this new way, but the old is still the old way ;-)